2023-11-16T00:23:59.5078774Z ##[group]Run actions/setup-python@v4
2023-11-16T00:23:59.5079275Z with:
2023-11-16T00:23:59.5079589Z   python-version: 3.8
2023-11-16T00:23:59.5080033Z   architecture: x64
2023-11-16T00:23:59.5080379Z   cache: pip
2023-11-16T00:23:59.5080700Z   check-latest: false
2023-11-16T00:23:59.5081305Z   token: ***
2023-11-16T00:23:59.5081619Z   update-environment: true
2023-11-16T00:23:59.5082016Z   allow-prereleases: false
2023-11-16T00:23:59.5082483Z ##[endgroup]
2023-11-16T00:23:59.6447256Z ##[group]Installed versions
2023-11-16T00:23:59.6520722Z Successfully set up CPython (3.8.18)
2023-11-16T00:23:59.6522235Z ##[endgroup]
2023-11-16T00:24:00.5551137Z [command]/opt/hostedtoolcache/Python/3.8.18/x64/bin/pip cache dir
2023-11-16T00:24:03.0749020Z /home/runner/.cache/pip
2023-11-16T00:24:03.7053485Z Received 6245959 of 6245959 (100.0%), 20.5 MBs/sec
2023-11-16T00:24:03.7055365Z Cache Size: ~6 MB (6245959 B)
2023-11-16T00:24:03.7085892Z [command]/usr/bin/tar -xf /home/runner/work/_temp/8049348c-6308-4699-89ee-dc32e3784776/cache.tzst -P -C /home/runner/work/GitHub_Stats_A/GitHub_Stats_A --use-compress-program unzstd
2023-11-16T00:24:03.7497585Z Cache restored successfully
2023-11-16T00:24:03.7523972Z Cache restored from key: setup-python-Linux-22.04-Ubuntu-python-3.8.18-pip-60f2a6c0212a9b74a155cd81b58dd07fd8b180833132566260d95f4de42fe218
